ParallelMatrixMultiplication/ParallelMatrixMultiplication/Matrix.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ParallelMultiplication
+{
```

Comment:
используйте file-scoped неймспейсы
```suggestion
namespace ParallelMultiplication;
```